File: 2023-elixir/test/day_13_test.exs
defmodule Day13Test do
  use ExUnit.Case

  import TestHelper
  import Day13

  test "part1" do
    assert part_1(input("day_13_example_1")) == 405
    assert part_1(input("day_13")) == 35538
  end

  test "part2" do
    # assert part_2(input("day_11")) == 613_686_987_427
  end
end

File: 2024/day13/day13_test.go
package day13

import (
	"testing"

	"github.com/stretchr/testify/assert"
)

func TestPart1(t *testing.T) {
	assert.Equal(t, 480, Part1("input_example_1"))
	assert.Equal(t, 28753, Part1("input"))
}

func TestPart2(t *testing.T) {
	assert.Equal(t, 102718967795500, Part2("input"))
}
